src/no/runsafe/framework/api/entity/animals/horses/IAbstractHorse.java

```diff
@@ -0,0 +1,19 @@
+package no.runsafe.framework.api.entity.animals.horses;
+
+import no.runsafe.framework.api.entity.IAnimal;
+import no.runsafe.framework.api.entity.ITameable;
+
+public interface IAbstractHorse extends IAnimal, ITameable
```

Comment:
In 1.11 and up horse types/variants are different classes and Horses extend AbstractHorse, I set it up that way to make it easier to update horses to 1.11 in the future.